src/docs/predicted_molecular_consequence.md

```diff
@@ -8,9 +8,81 @@ A set of information enabling interpretation of likely variant impact on genomic
 | feature_stable_id | string          | Feature (eg transcript) id
 | feature_type      | OntologyTermMetadata        | SO term for feature type (eg. transcript, regulatory feature)
 | consequences      | array of OntologyTermMetadata           | SO terms for predicted impact ( eg missense variants)
-| variant_representation            | array of VariantRepresentation or [ ]  | Nomenclature values and source
+| gene_stable_id | string| Gene stable id
+| gene_symbol | string | Gene symbol
+| variant_representation | array of VariantRepresentation or [ ]  | Nomenclature values and source
 | prediction_results | array of PredictionResult or [ ] | Scores from programs like SIFT which calculate transcript-specific deleteriousness scores
-| relative_locations | array of VariantRelativeLocation  | 
+| transcript_biotype | string |
+| cDNA_location | VariantRelativeLocation or null  | Relative location on cDNA
+| CDS_location | VariantRelativeLocation or null | Relative location on CDS
```

Comment:
I wouldn't use upper-case letters either for cDNA or for CDS. See https://github.com/Ensembl/ensembl-cdm-docs/blob/main/src/docs/product_generating_context.md